### ISSUE TITLE
Suggestion: A little cross on the upper-right corner of every thumbnail in Frequently Read screen

### ISSUE THREAD

**Issue by cheesecracker** [sumatrapdfreader/sumatrapdf#283]
A little suggestion for those thumbnails in Frequently Read screen: a little cross in the upper-right corner to let users remove it instead of **right-click, move the mouse down the context menu and choose Remove document**.  Just like in Firefox as shown in the image below.

![firefox-recent-pages-thumbnails-with-cross](https://cloud.githubusercontent.com/assets/5957892/9769939/6eadbee6-572d-11e5-81a8-cb5542e6f5f9.png)

This feature is not urgent.  We can wait :)

**Comment by weitzhandler** [comment]
Yes please. Having the Pin button is also very important. Thanks for everything!

**Comment by cheesecracker** [comment]
Issue #508 is NOT related to this bug.
